ecchronos-binary/src/bin/ecc-status.py

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+''''exec python -B -- "$0" ${1+"$@"} # '''
+# vi: syntax=python
+#
+# Copyright 2019 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from argparse import ArgumentParser
+import sys
+try:
+    from ecchronoslib import rest, table_formatter
+except ImportError:
+    sys.path.append('../pylib')
+    from ecchronoslib import rest, table_formatter
+
+
+def add_vnode_state_to_table(vnode_state, table):
+    entry = list()
+    entry.append(vnode_state.start_token)
+    entry.append(vnode_state.end_token)
+    entry.append(vnode_state.replicas)
```

Comment:
When printing status at table level, the involved replicas get an ugly formatting. Example:
[u'127.0.0.1', u'127.0.0.3']